ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -868,7 +902,7 @@ export function PreviewConcierge(props: PropsWithChildren<{}>) {
         // Changing the site will force-reload the iFrame and 'beforeunload'
         // event won't trigger withing; guest won't be submitting it's own checkout
         // in such cases.
-        dispatch(checkOutGuest());
+        dispatch(guestCheckOut());
```

Comment:
Missing payload. 
Does this even compile? It shouldn't. Check the action is typed accordingly.

Reply:
export const guestCheckOut = /*#__PURE__*/ createAction<{ path: string }>('GUEST_CHECK_OUT');
It looks good, and it is compiling and building. Not sure why it is not throwing error